Reflections/R5 - May 14.md

```diff
@@ -0,0 +1,14 @@
+## 1. What have you accomplished this week? (Please list each accomplishment, and explain)
+I've had a much more productive week this week then last week. I've implemented my zone and renown system as outlined in the previous reflection document. I've also done some minor tweaking on obj_toolbar. I have also done some user tests to get some more directed feedback on my project from various people not in the class.
+
+
+## 2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+The challenges I faced this week are also directly related to what I plan to accomplish in the following week. Namely I've had a bit of confusion understanding how I needed to document the testing process. I think I when I have gotten is a well enough type of documentation, but don't be surprised if it comes up a little bit rough.
```

Comment:
Are there any specific questions regarding testing you would like me to address? Feel free to reach out any time!